docs/input-jdbc.asciidoc

```diff
@@ -129,6 +129,9 @@ Here is the list:
 |sql_last_value | The value used to calculate which rows to query. Before any query is run,
 this is set to Thursday, 1 January 1970, or 0 if `use_column_value` is true and
 `tracking_column` is set. It is updated accordingly after subsequent queries are run.
+|offset, size| Values used in manual paging mode. Used in queries to explicitly implement
```

Comment:
Weird... github won't let me re-select those 3 rows so that I can make a suggestion.  So what about this: 

|offset, size| Values used with manual paging mode to explicitly implement
the paging. 
Supported only if <<plugins-{type}s-{plugin}-jdbc_paging_enabled>> and
<<plugins-{type}s-{plugin}-jdbc_paging_manual_mode>> are enabled.

I think it's OK not to repeat that this applies to queries as that is clearly stated in the lead-in sentence before the bullets.  That leaves us with a tighter description.   WDYT?

Reply:
Reads far better, I'll use it!